assets/stylesheets/application/_variables.scss

```diff
@@ -1,62 +1,9 @@
 // USWDS Overrides
```

Comment:
this file now just contains comments - does it need to exist?

Reply:
good question. yes, i anticipate using this as i further consolidate the css